lighthouse-plugin-publisher-ads/utils/network-timing.js

```diff
@@ -110,10 +111,55 @@ async function getTimingsByRecord(trace, devtoolsLog, context) {
   return timingsByRecord;
 }
 
+/**
+ * @param {LH.TraceEvent} e A trace event.
+ * @return {string|undefined} A script URL, if applicable.
+ */
+function getScriptUrl(e) {
+  if (!e.args.data) {
```

Comment:
!e.args || !e.args.data

Reply:
This is guaranteed to be non-nullable, or else it wouldn't pass type-checking